Hunting Queries/SecurityAlert/WebShellFileAlertEnrich.yaml

```diff
@@ -0,0 +1,40 @@
+id: d0a3cb7b-375e-402d-9827-adafe0ce386d
+name: Web shell file alert enrichment
+description: |
+  'Extracts MDATP Alert for a web shell being placed on the server and then enriches this event with information from W3CIISLog to idnetigy the Attacker that placed the shell'
+requiredDataConnectors:
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert
+  - connectorId: MicrosoftCloudAppSecurity
+    dataTypes:
+      - SecurityAlert
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+tactics:
+  - Privilege Escalation
+  - Persistence
+query: |
+  let timeWindow = 3d;
+  let scriptExtensions = dynamic([".php", ".jsp", ".js", ".aspx", ".asmx", ".asax", ".cfm", ".shtml"]);  
+  SecurityAlert  
+  | where TimeGenerated > ago(timeWindow)  
+  | where DisplayName =~ "Possible web shell installation"  
+  | extend alertData = parse_json(Entities)  
+  | mvexpand alertData  
+  // Get only the file type from the JSON, this gives us the file name
+  | where alertData.Type == "file"  
```

Comment:
use =~

Reply:
Change made, I can't believe this slipped the net!